### PR TITLE
feat!: next version

### PR DESCRIPTION
**Pull request by NikitaCG**

### CODE REVIEW

**Comment by stepastepastepa** [comment]
design review is ok

File: README.md [hunk 1]
```diff
@@ -60,14 +60,14 @@ import {
...
-To make sure the i18n library used in your project runs properly, perform its initialization and set the project's current locale value in `lang`. For example:
+The `blog-constructor` is a `uikit-based` library, and we use an instance of `i18n` from uikit. To set up internationalization, you just need to use the `configure` from uikit:
 
 ```typescript
-import {configure, Lang} from '@gravity-ui/blog-constructor';
+import {configure} from '@gravity-ui/uikit';
 
-configure({lang: Lang.En});
+configure({
+  lang: 'ru',
```
Comment (qradle-yndx) [inline README.md:69]: why not `Lang.Ru` from `@gravity-ui/uikit` Lang?

Comment (NikitaCG) [reply README.md:69]: I think it's copy-paste from page-constructor, a more accurate comment will give @niktverd

Comment (niktverd) [reply README.md:69]: yeap. It was like Nikita said

**Pull request merged**